docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/qovery.md

```diff
@@ -5,95 +5,70 @@ description: Learn in this guide how to deploy your Strapi application on Qovery
 
 ### 2. Create a project
-* Click on the **create a project** button and give a name to your project.
-* Click on **next**.
-
- 	::: tip NOTE
-	One project can have multiple apps running. This is convenient to group your backend, frontend, database etc.
-	:::
-
-### 3. Add your Strapi app
-* Click on the **create an application** button and select your Github or Gitlab repository where your Strapi app is located.
-* Click on **next**.
-
-### 4. Add a PostgreSQL database
-* Click on add a **PostgreSQL** database.
-* Select the version
-* Give a name to your PostgreSQL databse.
-* Click on **next**.
-
-### 5. Add a storage
-* Click on add a **Storage**.
-* Give a name to your storage.
-* Select the storage type between Slow HDD, HDD, SSD, and Fast SSD. (SSD is recommended)
-* Select the size.
-* Give a mount point.
-
-### 6. Deploy
-Click on the **Deploy** button. Your app should be deployed: you can see the status in real time by clicking on **Deployment logs**.
-
-## Deploying with the CLI
-### 1. Create a Qovery Account
-[Install the Qovery CLI](https://docs.qovery.com/docs/using-qovery/interface/cli/) and type the command `qovery auth` to create an account if you don't already have one.
-
-### 2. Add a .qovery.yml File
-Place a `.qovery.yml` file at the root of your project. Do not forget the dot when naming the file.
-
-```yaml
-application:
-  name: GIVE_A_NAME_TO_YOUR_APP
-  project: GIVE_A_PROJECT_NAME
-  publicly_accessible: true
-  storage: # qovery will attach automatically a SSD storage on /srv/app
-    - name: data
-      type: ssd
-      size: 5GB
-      mount_point: /srv/app
-databases:
-- type: postgresql # qovery will create a managed PostgreSQL database
-  version: 12
-  name: psql
-routers:
-- name: main
-  custom_domains: # optional: only if you want to use your domain
-  - branch: master
-    domain: my.domain.tld
-  routes:
-  - application_name: strapi
-    paths:
-    - /*
-```
-
-::: tip TIP
-For more information about the `.qovery.yml` file, please refer to [the Qovery application documentation](https://docs.qovery.com/docs/using-qovery/configuration/applications).
-:::
+* Click on **Create project** and give a name to your project.
+* Click on **Next**.
+
+![Create a project](https://hub.qovery.com/img/heroku/heroku-2.png)
 
 ::: tip NOTE
-Alternatively, you can deploy your Strapi application with a PostgreSQL database in the Qovery dashboard.
+One project can have multiple apps running. This is convenient to group your backend, frontend, database etc.
 :::
 
-### 3. Add a Dockerfile (optional)
+### 3. Create a new environment
+* Click on **Create environment** and give a name (e.g. staging, production).
+
+![Create a new environment](https://hub.qovery.com/img/heroku/heroku-3.png)
+
+### 4. Add your Strapi app
+* Click on **Create an application**, give a name and select your GitHub or GitLab repository where your Strapi app is located.
```

Comment:
These are steps performed in chronological order. So could you please replace bullet points with a numbered list? :-)

Reply:
```suggestion
* Click on **Create an application**, give a name and select your GitHub or GitLab repository where your Strapi application is located.
```